### DISCUSSION TITLE
Change table-zebra's row colors

### DISCUSSION THREAD

**Discussion by blockhunts** [saadeghi/daisyui#1049] [Q&A]
How to change row's looping colors in table-zebra class to, for example primary color?

Thank you 😄

**Comment by rhscjohn-dev** [comment]
I can change the text color but I could not get the background color to change.

I started off by removing `table-zebra`  from line 2.

on lines 14, 21 and 28 I added `class="odd:text-secondary even:text-primary"`.  This worked in alternating the color of the text.

I tried ` class="odd:bg-secondary even:bg-primary"` but that did not work.  Hopefully the developer can step in and explain why this did not work.  Or suggest a work around.

Here is a [Tailwind Play](https://play.tailwindcss.com/d5vCxXbdrD) showing alternat text color.

**Reply by blockhunts** [reply]
Thank you for the insight. 
Then I need to override the class manually for now
